use anyhow::Error;

pub async fn execute_status(name: &str) -> Result<(), Error> {
    todo!()
}
